build.gradle

```diff
@@ -55,6 +54,9 @@ allprojects {
         // Logging
         testRuntime group: 'ch.qos.logback', name: 'logback-classic', version: '1.1.7'
 
+	// Transitive dependencies
+        compile "com.typesafe.scala-logging:scala-logging_$scalaVersion:3.9.2"
```

Comment:
🤔 what changed here? why did we go from implementation to compile? I think we should use only `compile` and `api` so not sure what we're trying to do here?

Reply:
The `java-library` Gradle plugin has `implementation` and `api`. The `scala` Gradle plugin has `compile` and `implementation`. As the comment says I declared `scala-logging` as a transitive dependencies like Akka so that the consumer won't have to guess the Scala version etc.

Reply:
scala-logging is only used at compile time? I think there are some things we use at runtime also... such as the structured logging stuff

Reply:
> scala-logging is only used at compile time?

The name is misleading. It is a compile, runtime and transitive dependency. That means Projects depending on USI will pull in `scala-logging` as well.